src/webvitals.js

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {getCLS,getLCP,getFID} from 'web-vitals';
+
+function report(tracer, name, metric, reported) {
+  if (reported[name]) {
+    return;
+  }
+  reported[name] = true;
+  const value = metric.value;
+  const span = tracer.startSpan('webvitals');
+  span.setAttribute(name, value);
+  span.end(span.startTime);
```

Comment:
have we confirmed this with Justin? or are we deciding now it's going to be sent like that

Reply:
Excellent point.  I'll post on the team channel and seek some input.

Reply:
I was going to approve this, but now I'm not sure, I can't seem to find any conclusions in that Slack thread.

Reply:
I read it as "looks good enough" but we can wait another day or two for any more objections/discussion to pop up.